src/modelbench/benchmark_runner.py

```diff
@@ -115,7 +117,7 @@ def __init__(self, runner: "TestRunnerBase"):
         self.run_id = datetime.now().strftime("run-%Y%m%d-%H%M%S-%f")
         journal_dir = self.data_dir / "journals"
         journal_dir.mkdir(exist_ok=True, parents=True)
-        self.journal = RunJournal(journal_dir / f"journal-{self.run_id}.jsonl")
+        self.journal = RunJournal(journal_dir / f"journal-{self.run_id}.jsonl.zst")
```

Comment:
Modellab currently looks for a file named `run_{uid}_journal.jsonl`. It's trivial to change it there and I don't have a preference. Do you? @dhosterman WDYT?

Reply:
Unfortunately, the run_ids for the benchmarks and the journal are not consistent. Each benchmark result file gets its own run id, but the journal is for all the benchmarks (or no benchmarks; it works for test runs too).

I'm happy to resolve it any way that makes sense, but I'd rather merge this and do the change in a separate PR unless we can get a quick answer here.

Reply:
OK let's merge this and I will discuss with Daniel.